core/src/main/java/org/ehcache/core/spi/cache/AbstractValueHolder.java

```diff
@@ -181,6 +183,6 @@ public long getId() {
 
   @Override
   public String toString() {
-    return "" + value();
+    return format("%s", value());
```

Comment:
![This is sad](http://img.timeinc.net/time/photoessays/2010/halloween_costumes/sad_keanu.jpg)